src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -262,9 +407,13 @@ trait ModuleValue[T]
 
 object InModuleBody
 {
+  /** Code snippet injection at last of [[LazyModuleImp.instantiate]] in this [[LazyModule.scope]].
+    * It is used to create additional connections beside [[AutoBundle]] connection.
+    * */
   def apply[T](body: => T): ModuleValue[T] = {
```

Comment:
```suggestion
  /** Used to inject code snippets to be evaluated in  [[LazyModuleImp.instantiate]] in the current [[LazyModule.scope]].
    * It can be used to create additional hardware outside of the  [[LazyModule.children]] and
    * connections other than the internal [[BaseNode]] connections or additional IOs aside from the  [[AutoBundle]] 
    * */
  def apply[T](body: => T): ModuleValue[T] = {
```